odoo/tools/js_transpiler.py

```diff
@@ -513,14 +516,15 @@ def repl(matchobj):
 
 
 RELATIVE_REQUIRE_RE = re.compile(r"""
-    require\((?P<quote>["'`])([^@"'`]+)(?P=quote)\)  # require("some/path")
+    require\((?P<quote>["'`])([^"'`]+)(?P=quote)\)  # require("some/path")
     """, re.VERBOSE)
 
 
-def convert_relative_require(url, content):
+def convert_relative_require(url, dependencies, content):
```

Comment:
Should rename this function since it's now also extracting the dependencies. Either that or just make it a separate step, we're already running one regex per step anyway. If we want to optimize this entire thing to work in a single pass, it should be done as a concerted effort instead of just tacking on extra functionality to an existing step.

Reply:
ged prefers not to add a regex. What do you suggest for a name?

Reply:
i think the transpiler is sufficiently stand alone for now, no need to add an additional regexp step, nor refactoring it now. maybe later if/when we need to modify a feature